app/parsers/bulkrax/application_parser.rb

```diff
@@ -171,6 +173,28 @@ def valid_import?
       true
     end
 
+    # rubocop:disable Rails/SkipsModelValidations
+    def invalid_record(message)
+      current_run.invalid_records ||= ""
+      current_run.invalid_records += message
+      current_run.save
+      ImporterRun.find(current_run.id).increment!(:failed_records)
+      ImporterRun.find(current_run.id).decrement!(:enqueued_records) unless ImporterRun.find(current_run.id).enqueued_records <= 0 # rubocop:disable Style/IdenticalConditionalBranches
```

Comment:
I know this is still a draft, but putting a reminder to enable this cop again in this file.